README.md

```diff
@@ -6,82 +6,76 @@ This a node API wrapper for accessing the Trustpilot APIs. You can learn all abo
 
 ## Installation
 
-This module is built using Node.js `v4.0.x`.
+This module is built using Typescript and Node.js `v8.10.x`.
 
 If you are not using version 4 of Node, you'll have to transpile the code down to ES5 yourself.
 
 Install using `npm install trustpilot`
 
 ## Usage
 
-The trustpilot module is Promise based. It provides [request-promise](https://github.com/request/request-promise) objects with sane defaults.
+The trustpilot module is async/await based. It provides [request-promise-native](https://github.com/request/request-promise-native) objects with sane defaults.
 
 ### Basic Usage
 
 ```js
-let Trustpilot = require('trustpilot');
+import { TrustpilotApi } from "./trustpilot-api";
 
-let client = new Trustpilot(
-  {
+const client = new TrustpilotApi({
     apiKey: 'YOUR-API-KEY',
-    secret: 'YOUR-SECRET',
-    username: 'YOUR-TRUSTPILOT-B2B-USERNAME',
-    password: 'YOUR-TRUSTPILOT-B2B-PASSWORD'
+    apiSecret: 'YOUR-SECRET',
+    apiUsername: 'YOUR-TRUSTPILOT-B2B-USERNAME',
+    apiPassword: 'YOUR-TRUSTPILOT-B2B-PASSWORD'
   });
 
 // For basic calls authentified by API key, use client.apiRequest
-client.apiRequest('/v1/resources/images')
-  .then((response) => {
-    // handle the response
-  })
-  .catch((error) => {
-    // handle the error
-  });
+try {
```

Comment:
This example code wouldn't run if you copy/pasted it, since you cannot use `await` in module scope.

Move the example into a function or use a promise?